{"asctime": "2023-01-31 15:05:01,896", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 744, "module": "client_api", "threadName": "CP Server Thread-8", "message": "Authentication attempt invalid user: (user1)", "metric_name": "account.login.failed_invalid_user", "path_info": "/authenticate", "request_ip": "185.234.70.33, 185.234.70.33, 185.234.70.33, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:05:01.896185+00:00"}
{"asctime": "2023-01-31 15:05:05,305", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 744, "module": "client_api", "threadName": "CP Server Thread-13", "message": "Authentication attempt invalid user: (user2)", "metric_name": "account.login.failed_invalid_user", "path_info": "/authenticate", "request_ip": "185.234.70.33, 185.234.70.33, 185.234.70.33, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:05:05.305390+00:00"}
{"asctime": "2023-01-31 15:07:41,658", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 655, "module": "client_api", "threadName": "CP Server Thread-6", "message": "Authentication attempt invalid password for user: (user)", "metric_name": "account.login.failed_invalid_password", "path_info": "/authenticate", "request_ip": "192.166.246.231, 192.166.246.231, 192.166.246.231, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:07:41.658700+00:00"}
{"asctime": "2023-01-31 15:07:43,373", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 655, "module": "client_api", "threadName": "CP Server Thread-7", "message": "Authentication attempt invalid password for user: (user)", "metric_name": "account.login.failed_invalid_password", "path_info": "/authenticate", "request_ip": "192.166.246.231, 192.166.246.231, 192.166.246.231, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:07:43.373934+00:00"}
{"asctime": "2023-01-31 15:09:23,351", "name": "client_api_server", "processName": "MainProcess", "filename": "client_api.py", "funcName": "authenticate", "levelname": "WARNING", "lineno": 655, "module": "client_api", "threadName": "CP Server Thread-6", "message": "Authentication attempt invalid password for user: (user)", "metric_name": "account.login.failed_invalid_password", "path_info": "/authenticate", "request_ip": "185.234.70.44, 185.234.70.44, 185.234.70.44, 192.168.32.12", "user_agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10.15; rv:109.0) Gecko/20100101 Firefox/109.0", "timestamp": "2023-01-31T15:09:23.351967+00:00"}